Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx

```diff
@@ -111,6 +112,21 @@ export const TicketContainer = () => {
             </div>
           </div>
         </div>
+          <FormGroup>
+              <Label>Want to attach any files?</Label>
+              <FileUpload
+                  files={ticket.attachments || []}
+                  setFiles={(f) => setTicket({ ...ticket, attachments: [...f] })}
```

Comment:
Need to change this to using the lambda method like you did on the next line. Always use the lambda if you are going to use the state value in your setter.

Reply:
Ok, but I was just copying you code :)